src/react/components/common/keyboardBinding/keyboardBinding.tsx

```diff
@@ -0,0 +1,27 @@
+import { KeyboardContext, IKeyboardContext } from "../keyboardManager/keyboardManager";
+import React from "react";
+
+export interface IKeyboardBindingProps {
+    accelerator: string;
+    onKeyDown: (evt?: KeyboardEvent) => void;
+}
+
+export class KeyboardBinding extends React.Component<IKeyboardBindingProps> {
+    public static contextType = KeyboardContext;
+    public context!: IKeyboardContext;
```

Comment:
Is this supposed to be a question mark or does the bang operator do something here? If so, what is it doing?

Reply:
https://stackoverflow.com/questions/42273853/in-typescript-what-is-the-exclamation-mark-bang-operator-when-dereferenci